File: WebRoot/app/sysmodule/model/SysRole.js
Ext.define('srm.sysmodule.model.SysRole', {
	extend: 'Ext.data.Model',
	idProperty: 'role_id',
	identifier:'negative',
	fields: [
		{ name: 'role_id', type: 'int' },
		{ name: 'role_name' },
		{ name: 'role_desc' },
		{ name: 'order_seq', type: 'int' },
		{name:'is_enable',type:'int'},
		{name:'role_type',type:'int'}
	]
});

File: src/common/sysmodule/model/SysRole.java
package common.sysmodule.model;

import java.io.Serializable;

public class SysRole implements Serializable{
	/**
	 * 
	 */
	private static final long serialVersionUID = -8797242845565418707L;
	private int			role_id;
	private String			role_name;
	private String			role_desc;
	private int			order_seq;
	private int role_type;
	private int is_enable;
	public int getRole_id() {
		return role_id;
	}
	public void setRole_id(int role_id) {
		this.role_id = role_id;
	}
	public String getRole_name() {
		return role_name;
	}
	public void setRole_name(String role_name) {
		this.role_name = role_name;
	}
	public String getRole_desc() {
		return role_desc;
	}
	public void setRole_desc(String role_desc) {
		this.role_desc = role_desc;
	}
	public int getOrder_seq() {
		return order_seq;
	}
	public void setOrder_seq(int order_seq) {
		this.order_seq = order_seq;
	}
	public int getRole_type() {
		return role_type;
	}
	public void setRole_type(int role_type) {
		this.role_type = role_type;
	}
	public int getIs_enable() {
		return is_enable;
	}
	public void setIs_enable(int is_enable) {
		this.is_enable = is_enable;
	}
	
}
